cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -552,6 +578,8 @@ def make_resource(self, data, **kwargs):
             return ExistingFsxOpenZfs(**shared_volume_attributes)
         elif storage_type == FSX_ONTAP:
             return ExistingFsxOntap(**shared_volume_attributes)
+        elif storage_type == FSX_FILE_CACHE:
+            return ExistingFsxFileCache(**shared_volume_attributes)
```

Comment:
why do we need to inject shared_volume_attributes in File Cache?
In FileCache we do not have the concept of volume...

Reply:
shared_volume_attributes has mount_dir, Name, which are not Volume specific.

Reply:
*Observation to address in a separate PR, but worth starting the discussion for it.*

the variable `shared_volume_settings` should be renamed in shared_storage_settings` because not all shared storage has the notion of volumes (sometimes we have volumes, sometimes we have file systems, other times we have caches)